src/db_impl.cc

```diff
@@ -387,6 +389,7 @@ Status TitanDBImpl::CreateColumnFamilies(
   std::vector<ColumnFamilyDescriptor> base_descs;
   std::vector<std::shared_ptr<TableFactory>> base_table_factory;
   std::vector<std::shared_ptr<TitanTableFactory>> titan_table_factory;
+  std::vector<std::shared_ptr<BlobIndexMergeOperator>> merge_operators;
```

Comment:
Seem to me you can set the same merge operator object to all CFs. We can keep a shared_ptr at TitanDBImpl for reuse.

Reply:
Indeed, now the operator is completely stateless.